src/game-support/call-of-cthulhu-dcote.md

```diff
@@ -1,7 +1,7 @@
 # Call of Cthulhu: Dark Corners of the Earth
 <!-- script:Aliases [] -->
```

Comment:
IDK if you want to do this, but since you're formatting, can you add an alias?
`Call of Cthulhu Dark Corners of the Earth` (without the colon).